### PR TITLE
SSL Redirect rationalization

### PR DESCRIPTION
**Pull request by e0d**
@arbabnazar @jibsheet open to other ideas here, but think this is what is required.

### CODE REVIEW

**Comment by arbabnazar** [comment]
:+1: 
Please make change into the secure repo and refer to it.

**Comment by jibsheet** [comment]
I think this looks good, I'll update my secure branches :+1:

File: playbooks/roles/nginx/defaults/main.yml [hunk 1]
```diff
@@ -21,6 +21,10 @@ NGINX_REDIRECT_TO_HTTPS: False
 #
 # cat www.example.com.crt bundle.crt > www.example.com.chained.crt
 
+# This variable is only checked if NGINX_REDIRECT_TO_HTTPS is true
+# It should be set to one of !!null, "scheme" or "forward_for_proto"
+NGINX_HTTPS_REDIRECT_STRAGEGY: "scheme"
+
```
Comment (arbabnazar) [inline playbooks/roles/nginx/defaults/main.yml:27]: Please fix this typo before merging `STRATEGEY`

**Comment by arbabnazar** [comment]
@e0d can you please also add `NGINX_HTTPS_REDIRECT_STRATEGEY: "scheme"` inside the ansible-provision.sh? Thanks

**Comment by arbabnazar** [comment]
@e0d Thanks, please ignore my last comment, as you already addressed that. Thanks

**Comment by jibsheet** [comment]
This has now been deployed - we need to also hit the other files in sites-available/ for https redirects.  Secure changes should be fine, since I set those in the top-level edx.yml and edge.yml

**Comment by e0d** [comment]
@arbabnazar can you take over this branch and add fixes for the other IDAs as well as the cookie cutter template.

**Comment by arbabnazar** [comment]
https://github.com/edx/configuration/pull/2684

**Comment by jibsheet** [comment]
:+1:

**Pull request merged**